.github/ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,24 @@
-_Thanks for wanting to report an issue you've found in Node.js. Please delete
-this text and fill in the template below. If unsure about something, just do as
-best as you're able._
-
-_Note that it will be much easier for us to fix the issue if a test case that
-reproduces the problem is provided. Ideally this test case should not have any
-external dependencies. We understand that it is not always possible to reduce
-your code to a small test case, but we would appreciate to have as much data as
-possible. Thank you!_
-
-* **Version**: _output of `node -v`_
-* **Platform**: _either `uname -a` output, or if Windows, version and 32-bit or
-  64-bit_
-* **Subsystem**: _optional. if known - please specify affected core module name_
+<!--
+Thanks for wanting to report an issue you've found in Node.js. Please fill in
+the template below by replacing the html comments with an appropriate answer.
+If unsure about something, just do as best as you're able.
+
+It will be much easier for us to fix the issue if a test case that reproduces
+the problem is provided. Ideally this test case should not have any external
+dependencies. We understand that it is not always possible to reduce your code
+to a small test case, but we would appreciate to have as
+much data as possible.
+
+Thank you!
+
+-->
+
+* **Version**:   <!-- Output of `node -v` -->
+* **Platform**:  <!-- Either `uname -a` output, or if Windows, version and
+                      32-bit or 64-bit -->
+* **Subsystem**: <!-- Optional. If known - please specify
+                      affected core module name. -->
```

Comment:
GitHub doesn't appear to parse comments unless they are multiline from the start of the line? See: https://github.com/jbergstroem/node/blob/feature/improve-pr-commit-template/.github/ISSUE_TEMPLATE.md